src/core/hooks/useLocalStorage.ts

```diff
@@ -1,55 +1,57 @@
-import { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import { useCallback, useEffect, useState } from 'react';
 import { useCookies } from 'react-cookie';
-import useEventListener from './useEventListener';
 
-type SetValue<T> = Dispatch<SetStateAction<T>>;
-
-function useLocalStorage<T>(key: string, initialValue: T): [T, SetValue<T>] {
```

Comment:
Could be used https://github.com/DSpotDevelopers/makerdao-ses-dashboard/blob/master/src/core/utils/local-storage.ts in order to simplify the hook a bit?

Reply:
This will not need any more, this file is delete in the next PR, but I will check that code for use in the PR